moveit_core/kinematic_constraints/src/utils.cpp

```diff
@@ -227,6 +246,36 @@ moveit_msgs::msg::Constraints constructGoalConstraints(const std::string& link_n
   return goal;
 }
 
+bool updatePoseConstraint(moveit_msgs::msg::Constraints& constraints, const std::string& link_name,
```

Comment:
This function should just use something like `return updatePositionConstraint() && updateOrientationConstraint()`

Reply:
it's not quite that tidy b/c:

-  updatePositionConstraint() takes a geometry_msgs/Point instead of a geometry_msgs/Pose
- updateOrientationConstraint() takes a geometry_msgs::msg::QuaternionStamped instead of a geometry_msgs/Quaternion

So I'd have to convert the message types before calling those functions. I guess it's slightly cleaner that way